prettier.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  printWidth: 120,
+  singleQuote: true,
+  semi: false,
+  excludeFiles: 'fixtures/**/*-expected.js',
+  overrides: [
+    {
+      files: ['packages/**/src/**/*.js', 'examples/**/src/**/*.js'],
+      options: {
+        trailingComma: 'all',
```

Comment:
I think `trailingComma: 'all'` should move to the top level. ESLint starts to complains some configuration files (e.g. rollup.config.js) if I use prettier to format these files.

Reply:
good point, it's supported in trailing commas in functions are allowed since node@8 so we should be able to use it in all files